AsyncDisplayKit/ASVideoNode.mm

```diff
@@ -373,6 +407,7 @@ - (void)play
     }
   }
   
+  self.playerState = ASVideoNodePlayerStatePlaying;
```

Comment:
Btw please rebase from master first before making further changes to this PR. I think this diff is missing some key changes that has landed on master.